packages/expo-av/android/build.gradle

```diff
@@ -6,6 +6,9 @@ group = 'host.exp.exponent'
 version = '11.1.0'
 
 def REACT_NATIVE_DIR = new File(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim()).parent
+def RN_SO_DIR = findProject(":ReactAndroid")
+    ? Paths.get(findProject(":ReactAndroid").getProjectDir(), "build", "react-ndk", "exported")
```

Comment:
just a last nit for missing import for `Paths` from `import java.nio.file.Paths`. sorry not saying that clearly.